.gitignore

```diff
@@ -17,6 +17,7 @@ _x
 vck5000_aie_base_graph_hw.ltx
 vck5000_aie_base_graph_hw.xsa.link_summary
 Work
+sandbox
```

Comment:
Would be great to remove this, as virtualenv name doesn't have to be called "sandbox".

Reply:
Ah yes, good point. I will remove it!